### ISSUE TITLE
FireTV update killed this again.

### ISSUE THREAD

**Issue by yurividal** [BaronKiko/LauncherHijack#70]
FireTV updated last night, and lost this again

**Comment by oathzed** [comment]
> FireTV updated last night, and lost this again

Yeah I dunno how this is gonna get fixed. I've got accessibility service on for button mapper and the keys don't even register when I try to make them. Really weird. Fuckin amazon
